packages/runner-ct/src/iframe/iframes.jsx

```diff
@@ -18,28 +18,36 @@ export function getSpecUrl ({ namespace, spec }, prefix = '') {
 
 @observer
 export default class Iframes extends Component {
-  _disposers = []
-  containerRef = null
+  _disposers = [];
+  containerRef = null;
```

Comment:
can we remove the `;`? doesn't follow the code base style. we need a linter rule eventually